web/src/common/utils.ts

```diff
@@ -19,16 +19,11 @@ export function getCookie(name: string): string {
 }
 
 export function convertToSlug(text: string): string {
-    return text
-        .toLowerCase()
-        .replace(/ /g, "-")
-        .replace(/[^\w-]+/g, "");
+    return kebabCase(text);
```

Comment:
I dont think `kebabCase` does the second replace of removing anything but `\w` from the string (technically that should actually be \w\d)